File: app/src/main/java/com/hebat/kecamatanngaliyan/pesonangaliyanhebat/WisataActivity.java
package com.hebat.kecamatanngaliyan.pesonangaliyanhebat;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

import com.joooonho.SelectableRoundedImageView;

public class WisataActivity extends AppCompatActivity {

    SelectableRoundedImageView tamanlele, lembahKlp, indian, kapal, margasatwa, karangjoho, kaliancar, kalipancuran, serendeng;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_wisata);

        tamanlele = findViewById(R.id.tamanlele);
        tamanlele.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intentTamanLele = new Intent(WisataActivity.this, TamanLeleActivity.class);
                startActivity(intentTamanLele);
            }
        });

        lembahKlp = findViewById(R.id.lembahklp);
        lembahKlp.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intentLembahKlp = new Intent(WisataActivity.this, LembahKalipancurActivity.class);
                startActivity(intentLembahKlp);
            }
        });

        indian = findViewById(R.id.indian);
        indian.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intentIndian = new Intent(WisataActivity.this, IndianActivity.class);
                startActivity(intentIndian);
            }
        });

        kapal = findViewById(R.id.kapal);
        kapal.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intentKapal = new Intent(WisataActivity.this, KapalActivity.class);
                startActivity(intentKapal);
            }
        });

        margasatwa = findViewById(R.id.margasatwa);
        margasatwa.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intentMargasatwa = new Intent(WisataActivity.this, MargasatwaActivity.class);
                startActivity(intentMargasatwa);
            }
        });

        karangjoho = findViewById(R.id.karangjoho);
        karangjoho.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intentKarangjoho = new Intent(WisataActivity.this, KarangjohoActivity.class);
                startActivity(intentKarangjoho);
            }
        });

        kaliancar = findViewById(R.id.kaliancar);
        kaliancar.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intentKaliancar = new Intent(WisataActivity.this, KaliancarActivity.class);
                startActivity(intentKaliancar);
            }
        });

        kalipancuran = findViewById(R.id.kalipancuran);
        kalipancuran.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intentKalipancuran = new Intent(WisataActivity.this, KalipancuranActivity.class);
                startActivity(intentKalipancuran);
            }
        });

        serendeng = findViewById(R.id.serendeng);
        serendeng.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intentSerendeng = new Intent(WisataActivity.this, SerendengActivity.class);
                startActivity(intentSerendeng);
            }
        });
    }

    @Override
    public void onBackPressed() {
        Intent intentBack = new Intent(WisataActivity.this, MainActivity.class);
        startActivity(intentBack);
        finish();
    }
}

File: app/src/main/java/com/hebat/kecamatanngaliyan/pesonangaliyanhebat/WisataActivity.kt
package com.hebat.kecamatanngaliyan.pesonangaliyanhebat

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.joooonho.SelectableRoundedImageView

class WisataActivity : AppCompatActivity() {
    private var tamanlele: SelectableRoundedImageView? = null
    private var lembahKlp: SelectableRoundedImageView? = null
    private var indian: SelectableRoundedImageView? = null
    private var kapal: SelectableRoundedImageView? = null
    private var margasatwa: SelectableRoundedImageView? = null
    private var karangjoho: SelectableRoundedImageView? = null
    private var kaliancar: SelectableRoundedImageView? = null
    private var kalipancuran: SelectableRoundedImageView? = null
    private var serendeng: SelectableRoundedImageView? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_wisata)
        tamanlele = findViewById(R.id.tamanlele)
        tamanlele?.setOnClickListener {
            val intentTamanLele = Intent(this@WisataActivity, TamanLeleActivity::class.java)
            startActivity(intentTamanLele)
        }
        lembahKlp = findViewById(R.id.lembahklp)
        lembahKlp?.setOnClickListener {
            val intentLembahKlp = Intent(this@WisataActivity, LembahKalipancurActivity::class.java)
            startActivity(intentLembahKlp)
        }
        indian = findViewById(R.id.indian)
        indian?.setOnClickListener {
            val intentIndian = Intent(this@WisataActivity, IndianActivity::class.java)
            startActivity(intentIndian)
        }
        kapal = findViewById(R.id.kapal)
        kapal?.setOnClickListener {
            val intentKapal = Intent(this@WisataActivity, KapalActivity::class.java)
            startActivity(intentKapal)
        }
        margasatwa = findViewById(R.id.margasatwa)
        margasatwa?.setOnClickListener {
            val intentMargasatwa = Intent(this@WisataActivity, MargasatwaActivity::class.java)
            startActivity(intentMargasatwa)
        }
        karangjoho = findViewById(R.id.karangjoho)
        karangjoho?.setOnClickListener {
            val intentKarangjoho = Intent(this@WisataActivity, KarangjohoActivity::class.java)
            startActivity(intentKarangjoho)
        }
        kaliancar = findViewById(R.id.kaliancar)
        kaliancar?.setOnClickListener {
            val intentKaliancar = Intent(this@WisataActivity, KaliancarActivity::class.java)
            startActivity(intentKaliancar)
        }
        kalipancuran = findViewById(R.id.kalipancuran)
        kalipancuran?.setOnClickListener {
            val intentKalipancuran = Intent(this@WisataActivity, KalipancuranActivity::class.java)
            startActivity(intentKalipancuran)
        }
        serendeng = findViewById(R.id.serendeng)
        serendeng?.setOnClickListener {
            val intentSerendeng = Intent(this@WisataActivity, SerendengActivity::class.java)
            startActivity(intentSerendeng)
        }
    }

    override fun onBackPressed() {
        val intentBack = Intent(this@WisataActivity, MainActivity::class.java)
        startActivity(intentBack)
        finish()
    }
}